### PR TITLE
CHANGE(BMP-384): Modified edit JSON modal

### PR DESCRIPTION
**Pull request by FilippoRezzonico**
I changed the position of the buttons inside of the modal of the edit JSON button of api explorer. I also modified other properties of the modal and of the textarea inside of it.

[BMP-384](https://makeitapp.atlassian.net/browse/BMP-384)

- [x] tests are included
- [ ] the documentation is updated or integrated accordingly with your changes
- [ ] the changelog has been updated accordingly with your changes
- [X] you are not committing extraneous files or sensible data
- [X] added the link to the Jira task
- [X] commit message and branch name conventions

### CODE REVIEW

File: packages/api-explorer/src/JsonForm/configureJsonEditor.js [hunk 1]
```diff
@@ -10,8 +10,59 @@ function setDefaultCustomization (JSONEditor) {
...
+              removeClickListener()
+            }
+          }
+          const removeClickListener = () => {
+            document.removeEventListener('click', outsideClickListener)
+          }
+          document.addEventListener('click', outsideClickListener)
+          this.editjson_textarea.style.width = '450px'
+          this.editjson_textarea.style.height = '340px'
+          this.editjson_holder.style.left = `${parseInt(this.editjson_holder.style.left, 10) -this.editjson_holder.offsetWidth - parseInt(this.editjson_control.style.marginLeft, 10)}px`
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/JsonForm/configureJsonEditor.js:28]: If `this.editjson_holder.style.left` is undefined this line will generate NaN.

I'd calculate the size before, checking if it's safe and then set it

File: packages/api-explorer/src/JsonForm/configureJsonEditor.js [hunk 2]
```diff
@@ -10,8 +10,59 @@ function setDefaultCustomization (JSONEditor) {
...
+          this.editjson_holder.style.top = `-104px`
+        }
+      }
       postBuild() {
         super.postBuild()
+        if (this.editjson_holder && this.editjson_textarea && this.editjson_copy && this.editjson_save && this.editjson_cancel) {
+          this.editjson_inline_header = document.createElement('div')
+          this.editjson_inline_header.style.display = 'flex'
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/JsonForm/configureJsonEditor.js:36]: Can't you do:

```
this.editjson_inline_header.style = {
  display: 'flex',
  ...
}
```

File: packages/api-explorer/src/JsonForm/configureJsonEditor.js (hunk 1 above)
Comment (FilippoRezzonico) [reply packages/api-explorer/src/JsonForm/configureJsonEditor.js]: Check added!

File: packages/api-explorer/src/JsonForm/configureJsonEditor.js (hunk 2 above)
Comment (FilippoRezzonico) [reply packages/api-explorer/src/JsonForm/configureJsonEditor.js:39]: I tried, but actually style properties are not passed correctly to elements if done in that way!

**Pull request merged**